app/src/main/java/com/example/simbirsoft_java_core_training/education/Student.java

```diff
@@ -12,7 +12,14 @@ public class Student {
     private int groupNumber;
     private Map<String, Integer> grades;
 
-    public Student(int id, String lastName, String firstName, String middleName, int birthYear, int course, int groupNumber, Map<String, Integer> grades) {
+    public Student(int id,
+                   String lastName,
+                   String firstName,
+                   String middleName,
+                   int birthYear,
+                   int course,
+                   int groupNumber,
+                   Map<String, Integer> grades) {
```

Comment:
Плохое форматирование. На самом деле - дело вкуса программиста или кодстайла проекта, но сейчас мода на то, что если открывающая и закрывающая скобка не уместились на одной строке, то они должны быть соответственно последним и первым символом своих строк:
```suggestion
    public Student(
                   int id,
                   String lastName,
                   String firstName,
                   String middleName,
                   int birthYear,
                   int course,
                   int groupNumber,
                   Map<String, Integer> grades
               ) {
```

Reply:
И вообще, как-то код стрёмно выглядит. Бахни по нему автоформатированием на всякий случай